### PR TITLE
Add Croatian translation

### PR DESCRIPTION
**Pull request by kjozic**

### CODE REVIEW

File: i18n/hr.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  translation: Stranica
+
+- id: previous
+  translation: Prethodna
+
+- id: next
+  translation: Slijedeća
+
+- id: author_by
+  translation: autor
```
Comment (osg) [inline i18n/hr.yaml]: I assume this is a translation workaround for a troublesome "by". Is that a correct assumption?

Comment (osg) [reply i18n/hr.yaml]: Nearly completely ignorant to the grammatical nuances of Croatian, so bear with my comment.

**Comment by osg** [comment]
FWIW, I reviewed each line, and have a high level of confidence given the use of Google Translate and the English source that the translations are accurate enough for merging this PR.

**Comment by osg** [comment]
A native speaker came back with this comment:

"Yes, it looks fine, only one small mistake: Slijedeća -> Sljedeća (without "i"). People make this mistake very frequently."

File: i18n/hr.yaml (hunk 1 above)
Comment (kjozic) [reply i18n/hr.yaml]: Can you give me an example of a whole sentence so I can check the translation?

Comment (osg) [reply i18n/hr.yaml]: Not 100% sure. Perhaps it is in this string "By Ursula Kallio / 2019-12-17 / In categories Posts" at https://infobits.org/en/asynchronous-online-teaching/?

I am not sure if the initially is being forced as an uppercase letter or not without more investigation.

Comment (osg) [reply i18n/hr.yaml]: Yep, it’s here: https://github.com/shaform/hugo-theme-den/blob/master/i18n/en.yaml#L69
In English, the term is simply "By".

Comment (kjozic) [reply i18n/hr.yaml]: In this case, the translation is correct.

**Pull request merged**